src/components/tasks/crud/createTask.js

```diff
@@ -4,13 +4,14 @@ import {AddDescription} from "./description/AddDescription";
 import {convertFileToBase64} from "../../../utils/apis/files";
 import {AddTests} from "./tests/AddTests";
 import {AddDeadline} from "./AddDeadline";
-import {tasksApi} from "../../../utils/apis/tasksAPI";
+import {withSwal} from "../../commons/formsCommons/WithSwal";
+import React from "react";
 
 let task = {}
 
 const clearTask = () => task = {}
 
-const handleMarkdownChange = ({html, text}) => {
+const handleMarkdownChange = ({text}) => {
```

Comment:
Sprawdzałeś, że to wszystko wciąż działa po zmianach, nie?

Reply:
Tak, w sumie tu się nic nie zmieniło